tests/integration/distribution/keeper/delegation_test.go

```diff
@@ -1,820 +0,0 @@
-package keeper_test
```

Comment:
we should make sure to carry over some of these tests if they arent handled anywhere else

Reply:
these tests are already being tested in `x/distr` module using mocks. So I think we don't need to have them here.